packages/core/strapi/bin/strapi.js

```diff
@@ -337,6 +338,14 @@ program
   )
   .arguments('<filename>')
   .allowExcessArguments(false)
+  .hook(
+    'preAction',
+    confirmKeyValue(
```

Comment:
Wow, that's neat!

Reply:
It is, though I actually think it could be better in a few ways but it's good enough for this purpose and we can clean it up when we refactor the whole CLI in a few months.